libs/application/templates/general-petition/src/lib/messages.ts

```diff
@@ -0,0 +1,620 @@
+import { defineMessages, MessageDescriptor } from 'react-intl'
+
+type MessageDir = Record<string, Record<string, MessageDescriptor>>
+
+export const m: MessageDir = {
+  // Messages shared across party letter application
+  externalDataSection: defineMessages({
+    title: {
+      id: 'gpt.application:terms.title',
+      defaultMessage: 'Skilmálar',
+      description: 'External data title',
+    },
+    subtitle: {
+      id: 'gpt.application:terms.subtitle',
+      defaultMessage: 'Eftirfarandi gildir um söfnun meðmæla',
+      description: 'External data subtitle',
+    },
+    dmrTitle: {
+      id: 'gpt.application:dmr.title',
+      defaultMessage: 'Dómsmálaráðuneyti',
+      description: 'External data DRM title',
+    },
+    dmrSubtitle: {
+      id: 'gpt.application:dmr.subtitle',
+      defaultMessage:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi vestibulum auctor ipsum. Sed at erat odio. In consectetur sem bibendum augue gravida consectetur. Interdum et malesuada fames ac ante ipsum primis in faucibus. Aliquam fringilla sem a quam pellentesque, eu condimentum augue vestibulum. Integer porttitor ante nec tellus accumsan, vel cursus velit commodo. Cras pellentesque lacus non est interdum volutpat.',
```

Comment:
Lorem Ipsum

Reply:
we dont have the correct terms and conditions at this point, will update asap